tests/src/main/java/com/hazelcast/stabilizer/tests/synthetic/SyntheticTest.java

```diff
@@ -207,18 +211,18 @@ public void doRun() throws Exception {
             operations.addAndGet(iteration % performanceUpdateFrequency);
         }
 
-        private ICompletableFuture invoke(int partitionId) throws Exception {
-            ICompletableFuture f;
+        private ICompletableFuture<SyntheticRequest> invoke(int partitionId) throws Exception {
```

Comment:
Perhaps I'm missing something, but why is the future returning something of a particular type? The operation always returns null. SyntheticRequest is certainly not something returned as a response.